docs/sdks/server-sdks/dotnet.md

```diff
@@ -49,60 +75,262 @@ internal class AssignmentLogger : IAssignmentLogger
+         ["from"] = "usa"
+     },
+     ["adidas"] = new Dictionary<string, object?>()
+     {
+         ["brandLoyalty"] = 2,
+         ["from"] = "germany"
+     }
+ };
+ var result = client.GetBanditAction(
+     "flag-with-shoe-bandit",
+     "user123",
+     subjectAttributes,
+     actions,
+     "default");
+
+if (result.Action != null)
+{
+    // Follow the Bandit action
+    renderShoeAd(result.Action);
+} else {
+    // User was not selected for a Bandit.
+    // A variation is still assigned.
+    renderDefaultShoeAd(result.Variation);
+}
 ```
 
-### Handling the null assignment
 
-We recommend always handling the empty assignment case in your code. Here are some examples illustrating when the SDK returns `null`:
+## Full Initialization and Assignment Example
 
-1. The **Traffic Exposure** setting on experiments/allocations determines the percentage of subjects the SDK will assign to that experiment/allocation. For example, if Traffic Exposure is 25%, the SDK will assign a variation for 25% of subjects and `null` for the remaining 75% (unless the subject is part of an allow list).
+```cs
+class Program
+{
+    public void main()
+    {
 
-2. Assignments occur within the environments of feature flags. You must enable the environment corresponding to the feature flag's allocation in the user interface before `getStringAssignment` returns variations. It will return `null` if the environment is not enabled.
+        // Initialize Segment and Eppo clients.
+        var segmentConfig = new Configuration(
+                    "<YOUR WRITE KEY>",
+                    flushAt: 20,
+                    flushInterval: 30);
+        var analytics = new Analytics(segmentConfig);
+
+        // Create a logger to send data back to the Segment data warehouse
+        var logger = new SegmentLogger(analytics);
+
+        // Initialize the Eppo Client
+        var eppoClientConfig = new EppoClientConfig("EPPO-SDK-KEY-FROM-DASHBOARD", logger);
+        var eppoClient = EppoClient.Init(eppoClientConfig);
+
+        // Elsewhere in your code, typically just after the user logs in.
+        var subjectTraits = new JsonObject()
+        {
+            ["email"] = "janedoe@liamg.com",
+            ["age"] = 35,
+            ["accountAge"] = 2,
+            ["tier"] = "gold"
+        }; // User properties will come from your database/login service etc.
+        var userID = "user-123";
+
+        // Identify the user in Segment Analytics.
+        analytics.Identify(userID, subjectTraits);
+
+
+        // Need to reformat user attributes a bit; EppoClient requires `IDictionary<string, object?>`
+        var subjectAttributes = subjectTraits.Keys.ToDictionary(key => key, key => (object)subjectTraits[key]);
+        // Get an assignment for the user
+        var assignedVariation = eppoClient.GetStringAssignment(
+            "new-user-onboarding",
+            userID,
+            subjectAttributes,
+            "control"
+        );
+    }
```

Comment:
what about also getting a bandit assignment after this? (if so since that is a string typed method, consider changing the above to something else like a boolean)